src/test/resources/insertUsers.graphql

```diff
@@ -0,0 +1,9 @@
+mutation insert_users($id: uuid!, $name: String!, $rocket: String!) {
+    insert_users(objects: {id: $id, name: $name, rocket: $rocket}) {
+        returning {
+            id
+            name
+            rocket
+        }
+    }
+}
```

Comment:
Saznao sam skoro da ipak postoji razlog sto se ostavlja jedna prazna linija u svakom java file-u. Ako sam ti inicijalno sugerisao da to ne radis; zanemari to - u pitanju je moja greska